### PR TITLE
Add release notes for Deploy 13.1.1 and 14.0.2

### PR DESCRIPTION
**Pull request by ronaldbarendse**
## Description

Add release notes for Deploy 13.1.1 and 14.0.2.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [x] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Deploy 13.1.1 and 14.0.2.

## Deadline (if relevant)

Thursday July 11th (the patches are already released to NuGet)

### CODE REVIEW

File: 14/umbraco-deploy/getting-started/deploy-settings.md [hunk 1]
```diff
@@ -180,6 +181,12 @@ If encountering this issue, there are two batch settings that can be applied wit
...
 * `PackageBatchSize` - applies a batch setting to the processing of a Deploy "package", which contains all the items selected for a Deploy operation, plus all the determined dependencies and relations.
 
+### MaxRequestLength
+
+When Deploy transfers files between environments that can't connect to each other (like a local environment that isn't exposed publicly), it falls back from fetching/downloading the files to uploading them. This is done in fixed-sized chunks, so multiple files can be combined in a single request or a large file into multiple requests. This setting can specify the size of these chuncks **(in bytes)**.
+
+If no value is set, Deploy will create 16MB chunks, unless the [CMS `MaxRequestLength` setting](../../umbraco-cms/reference/configuration/runtimesettings.md) is set to a lower value (note that the CMS setting is configured in kilobytes). Setting this to a value higher than the CMS `MaxRequestLength` (multiplied by 1024) will cause a validation error. Similarly, configuring a value that is higher than the [server or infrastructure allows](../../umbraco-cms/reference/configuration/maximumuploadsizesettings.md) will result in file transfers failing.
```
Comment (AndyButland) [inline 14/umbraco-deploy/getting-started/deploy-settings.md:188]: IIRC these relative links across "sites" within the documentation won't work, and instead you'll need to create an absolute HTML link to the published page on the docs site when linking to CMS documentation.

File: 14/umbraco-deploy/getting-started/deploy-settings.md [hunk 2]
```diff
@@ -180,6 +181,12 @@ If encountering this issue, there are two batch settings that can be applied wit
 * `SourceDeployBatchSize` - applies a batch setting for the transfer of multiple selected items to an upstream environment (such as a media folder with many images).
 * `PackageBatchSize` - applies a batch setting to the processing of a Deploy "package", which contains all the items selected for a Deploy operation, plus all the determined dependencies and relations.
 
+### MaxRequestLength
+
+When Deploy transfers files between environments that can't connect to each other (like a local environment that isn't exposed publicly), it falls back from fetching/downloading the files to uploading them. This is done in fixed-sized chunks, so multiple files can be combined in a single request or a large file into multiple requests. This setting can specify the size of these chuncks **(in bytes)**.
```
Comment (AndyButland) [inline 14/umbraco-deploy/getting-started/deploy-settings.md:186]: Typo above "chuncks" --> "chunks"

File: 14/umbraco-deploy/getting-started/deploy-settings.md [hunk 3]
```diff
@@ -180,6 +181,12 @@ If encountering this issue, there are two batch settings that can be applied wit
 * `SourceDeployBatchSize` - applies a batch setting for the transfer of multiple selected items to an upstream environment (such as a media folder with many images).
 * `PackageBatchSize` - applies a batch setting to the processing of a Deploy "package", which contains all the items selected for a Deploy operation, plus all the determined dependencies and relations.
 
+### MaxRequestLength
```
Comment (AndyButland) [inline 14/umbraco-deploy/getting-started/deploy-settings.md:184]: Looks like you haven't added this to the 13 version of this page.  Assuming it's still valid for that version, it should be added there too.

**Comment by eshanrnh** [comment]
Thanks, @ronaldbarendse for the PR 🙌 We will review it as soon as we can. 💪

File: 13/umbraco-deploy/release-notes.md [hunk 4]
```diff
@@ -17,6 +17,13 @@ If you are upgrading to a new major version you can find the details about the b
 
 This section contains the release notes for Umbraco Deploy 13 including all changes for this version.
 
+#### [13.1.1](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.1.1) (July 11th 2024)
+
+* Add `[DisableRequestSizeLimit]` attribute to `UploadForImport` endpoint to remove application request size limit (server/infrastructure restrictions may still apply)
+* Determine missing `routePath` based on `$routeParams` to fix Export, Queue for transfer and Partial Restore actions on items with children in a list view
```
Comment (eshanrnh) [inline 13/umbraco-deploy/release-notes.md:23]:
```suggestion
* Determine missing `routePath` based on `$routeParams` to fix Export, Queue for transfer, and Partial Restore actions on items with children in a list view
```

File: 13/umbraco-deploy/release-notes.md [hunk 5]
```diff
@@ -17,6 +17,13 @@ If you are upgrading to a new major version you can find the details about the b
...
+* Set trashed state when processing content (https://github.com/umbraco/Umbraco.Deploy.Issues/issues/223)
```
Comment (eshanrnh) [inline 13/umbraco-deploy/release-notes.md:24]:
```suggestion
* Set trashed state when processing content [223](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/223)
```

File: 13/umbraco-deploy/release-notes.md [hunk 6]
```diff
@@ -17,6 +17,13 @@ If you are upgrading to a new major version you can find the details about the b
...
+* Improve exception message when parent can't be found when getting artifact (https://github.com/umbraco/Umbraco.Deploy.Issues/issues/216)
```
Comment (eshanrnh) [inline 13/umbraco-deploy/release-notes.md:25]:
```suggestion
* Improve exception message when parent cannot be found when getting artifact [216](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/216)
```

File: 14/umbraco-deploy/getting-started/deploy-settings.md (hunk 2 above)
Comment (eshanrnh) [inline 14/umbraco-deploy/getting-started/deploy-settings.md:186]:
```suggestion
When Deploy transfers files between unconnected environments (like a non-public local environment), it falls back from fetching/downloading the files to uploading them. This is done in fixed-sized chunks, so multiple files can be combined in a single request or a large file into multiple requests. This setting can specify the size of these chunks **(in bytes)**.
```

File: 14/umbraco-deploy/getting-started/deploy-settings.md (hunk 1 above)
Comment (eshanrnh) [inline 14/umbraco-deploy/getting-started/deploy-settings.md:188]:
```suggestion
If no value is set, Deploy will create 16MB chunks, unless the [CMS `MaxRequestLength` setting](../../umbraco-cms/reference/configuration/runtimesettings.md) is lower (CMS setting is configured in kilobytes). Setting a value higher than the CMS `MaxRequestLength` (multiplied by 1024) will cause a validation error. Similarly, configuring a value higher than the [server or infrastructure limits](../../umbraco-cms/reference/configuration/maximumuploadsizesettings.md) will result in file transfers failures.
```

File: 14/umbraco-deploy/release-notes.md [hunk 7]
```diff
@@ -18,6 +18,13 @@ If you are upgrading to a new major version you can find the details about the b
 
 This section contains the release notes for Umbraco Deploy 13 including all changes for this version.
 
+#### [14.0.2](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F14.0.2) (July 11th 2024)
+
+* Set trashed state when processing content (https://github.com/umbraco/Umbraco.Deploy.Issues/issues/223)
+* Improve exception message when parent can't be found when getting artifact (https://github.com/umbraco/Umbraco.Deploy.Issues/issues/216)
+* Add `MaxRequestLength` Deploy setting to break file upload into multiple requests
+* Set variant names when creating new content (https://github.com/umbraco/Umbraco.Deploy.Issues/issues/222)
```
Comment (eshanrnh) [inline 14/umbraco-deploy/release-notes.md:26]:
```suggestion
* Set trashed state when processing content [223](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/223)
* Improve exception message when parent can't be found when getting artifact [216](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/216)
* Add `MaxRequestLength` Deploy setting to break file upload into multiple requests
* Set variant names when creating new content [222](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/222)
```

**Review by eshanrnh — COMMENTED** [review]
I've added some suggestions to shorten the sentences and fix the vale errors. Can you take a look and see that it doesn't change the sentence completely🙂 

I'll incorporate the minor suggestions in the PR 💪

File: 13/umbraco-deploy/release-notes.md [hunk 8]
```diff
@@ -17,6 +17,13 @@ If you are upgrading to a new major version you can find the details about the b
 
 This section contains the release notes for Umbraco Deploy 13 including all changes for this version.
 
+#### [13.1.1](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.1.1) (July 11th 2024)
+
+* Add `[DisableRequestSizeLimit]` attribute to `UploadForImport` endpoint to remove application request size limit (server/infrastructure restrictions may still apply)
+* Determine missing `routePath` based on `$routeParams` to fix Export, Queue for transfer, and Partial Restore actions on items with children in a list view
+* Set trashed state when processing content [223](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/223)
+* Improve exception message when parent cannot be found when getting artifact [216](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/216)
```
Comment (eshanrnh) [inline 13/umbraco-deploy/release-notes.md:25]:
```suggestion
* Set trashed state when processing content [#223](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/223)
* Improve exception message when parent cannot be found when getting artifact [#216](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/216)
```

File: 14/umbraco-deploy/release-notes.md [hunk 9]
```diff
@@ -18,6 +18,13 @@ If you are upgrading to a new major version you can find the details about the b
 
 This section contains the release notes for Umbraco Deploy 13 including all changes for this version.
 
+#### [14.0.2](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F14.0.2) (July 11th 2024)
+
+* Set trashed state when processing content [223](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/223)
+* Improve exception message when parent can't be found when getting artifact [216](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/216)
+* Add `MaxRequestLength` Deploy setting to break file upload into multiple requests
+* Set variant names when creating new content [222](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/222)
```
Comment (eshanrnh) [inline 14/umbraco-deploy/release-notes.md:26]:
```suggestion
* Set trashed state when processing content [#223](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/223)
* Improve exception message when parent can't be found when getting artifact [#216](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/216)
* Add `MaxRequestLength` Deploy setting to break file upload into multiple requests
* Set variant names when creating new content [#222](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/222)
```

File: 14/umbraco-deploy/getting-started/deploy-settings.md (hunk 3 above)
Comment (ronaldbarendse) [reply 14/umbraco-deploy/getting-started/deploy-settings.md:184]: This setting is only introduced in v14 after refactoring the endpoints, so would require backporting first. Might be something to consider, although it's not been something users have requested or caused any issues in the past.

**Comment by eshanrnh** [comment]
Looks like this PR is good to be merged in 🙌

**Pull request merged**